src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --header-height: 60px;
+}
```

Comment:
Don't clear margins and paddings using *, it's resource-heavy operation.
* should be used only for box-sizing.
* 
variables are better to put into :root element
https://developer.mozilla.org/en-US/docs/Web/CSS/:root

And fonts - to `html` and inherited where necessary.